middleware/logger/logger.go

```diff
@@ -78,6 +78,24 @@ func New(config ...Config) fiber.Handler {
 	// Check if format contains latency
 	cfg.enableLatency = strings.Contains(cfg.Format, "${latency}")
 
+	// Add TagStatus Config
+	tagMap[TagStatus] = func(buf *bytebufferpool.ByteBuffer, c *fiber.Ctx, w io.Writer, tag string) (int, error) {
```

Comment:
can you please move all the definitions of the tag map into the tag file

maybe in one big function with the configuration of the user 

Then you could also move these tag functions directly into the tag map that the user has within the configuration so that collisions with other configurations for other logging middlewares don't happen

Reply:
ok, i will try